src/daemon/CMakeLists.txt

```diff
@@ -396,6 +412,23 @@ elseif (LINUX)
         endif ()
     endif ()
 
+    # If packaging, emit the the system manager configuration files so they can be setup post-install
+    if (PACKAGING)
```

Comment:
What is the purpose of this? Why is service installation in postinst in the first place? Shouldn't only activation be in postinst?